client/test/unit/specs/transactions.spec.js

```diff
@@ -166,53 +196,99 @@ describe('Testing Transactions.vue', () => {
         stateCd: 'INPROGRESS',
         user_action: 'Reset',
         user_name: 'github/cameronidir',
-      }]
-  })
+      }
+    ])
 
-  test('It displays the no transaction available message', async () => {
-    data.vm.$store.state.pendingTransactionsRequest = false
-    data.vm.$store.state.transactionsData = []
+    vm.$store.commit('setPendingTransactionsRequest', false)
+  })
 
-    await Vue.nextTick()
+  it('displays the no transaction available message', async () => {
+    await vm.$store.commit('setTransactionsData', [])
```

Comment:
Interesting: this only works while awaited, or with a delay after it. So it seems `commit` is async.